templates/confidential-computing/index.html

```diff
@@ -339,26 +365,27 @@ <h2>Confidential computing in your private data centre</h2>
             on-premises servers, using hardware-rooted primitives beyond
             traditional measures.
           </p>
+          <hr class="p-rule--muted" />
```

Comment:
Won't let me leave the suggestion directly, but lines 368-373 can be refactored with the following:
```
          <div class="p-cta-block">
            <a href="/pro/subscribe" class="p-button">Get a free trial for Ubuntu Pro</a>
            <a href="https://canonical.com/blog/why-do-you-also-need-confidential-computing-for-your-private-datacenter">Why use confidential VMs in your private datacenter?&nbsp;&rsaquo;</a>
          </div>
```